src/models/HooksServiceConfig.js

```diff
@@ -25,6 +26,17 @@ module.exports = BaseConfig.extend({
     webhookEventsQueueName: {
       description: 'The name of the queue in which webhook events are published on',
       default: 'events'
+    },
+    githubWebhookSecret: {
+      type: String,
+      description: 'path to file containing a shared secret for verifying github payloads',
+      default: function () {
+        let path
+        try {
+          path = require.resolve('~/.secrets/github-hooks-secret')
+        } catch (err) {}
+        return path ? fs.readFileSync(path, 'utf8').trim() : 'superSecret'
```

Comment:
I think instead of setting a default, we should throw an error if there is none.